frontend/src/ShareButton/ShareButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { ShareIcon } from "@heroicons/react/24/outline";
```

Comment:
⚠️ action required: file should not be needed. but thank you for your hard work here!!